comet-java-client/src/test/java/ml/comet/experiment/impl/OnlineExperimentTest.java

```diff
@@ -445,22 +448,66 @@ public void testUploadAndGetAssets() {
     }
 
     @Test
-    public void testSetsContext() {
-        OnlineExperiment experiment = createOnlineExperiment();
+    public void testLogAndGetRemoteAssets() {
+        try (OnlineExperimentImpl experiment = (OnlineExperimentImpl) createOnlineExperiment()) {
+            // Make sure experiment has no assets
+            //
+            assertTrue(experiment.getAssetList(ASSET_TYPE_ALL).isEmpty());
+
+            // Log remote assets and wait for completion
+            //
+            ExperimentContext context = new ExperimentContext(10, 101, "train");
+            OnCompleteAction onComplete = new OnCompleteAction();
+            Map<String, Object> metadata = new HashMap<String, Object>() {{
+                put("someInt", 10);
+                put("someString", "test string");
+                put("someBoolean", true);
+            }};
+            String metadataJson = JsonUtils.toJson(metadata);
```

Comment:
this is used for assertion only, move inside the assertion function, pass the actual map.

Reply:
Fixed.